wms/models/datasets/ugrid.py

```diff
@@ -284,7 +281,29 @@ def getmap(self, layer, request):
                     return self.empty_response(layer, request)
 
                 if request.GET['image_type'] == 'filledcontours':
-                    return mpl_handler.tricontourf_response(tri_subset, data, request)
+                    if np.isnan(np.sum(data)):
```

Comment:
Why not: `if np.isnan(data).any()`?  Clearer than `np.sum()`, unless there is a performance difference there...